doc/HACKING/WritingTests.md

```diff
@@ -7,8 +7,8 @@ keep from introducing bugs.  The major ones are:
 
    1. Unit tests written in C and shipped with the Tor distribution.
 
-   2. Integration tests written in Python and shipped with the Tor
-      distribution.
+   2. Integration tests written in Python (>= 2.7) and shipped with
```

Comment:
Should also mention that python3 needs to be 3.1 or newer.

Reply:
Fixed - b87b175a21f3ce8b6fa7f2faf22d1d1c5a338484.